docs/alerts.md

```diff
@@ -20,10 +20,82 @@
 ## Elasticsearch Cluster Health is Red
 
 At least one primary shard and its replicas are not allocated to a node.
+The most common causes can be failed cluster nodes and the elasticsearch process crashing due to a continuous heavy processing load.
 
 ### Troubleshooting
 
-Cluster nodes could be failed or the Elasticsearch process crashes due to heavy load.
+1. Confirm if the elasticsearch cluster’s health is RED:
+   ```
+   oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_cluster_health?pretty
+   ```
+   Check for the “status” field in the output of the above command.
+
+2. Check if all nodes have rejoined cluster:
+   ```
+   oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_cluster_health
+   ```
+   Check for the "number_of_nodes" field to get the total number of nodes allocated to the cluster.
+   Take a count of how many nodes have joined the cluster:
+   ```
+   oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_cat/nodes?v
```

Comment:
we can get the same information from either of these commands... either we use `es_cluster_health` or look at the output of the `_cat/nodes` command.